components/Team.tsx

```diff
@@ -0,0 +1,41 @@
+import team from 'data/team.json';
+import Image from 'next/image';
+
+export const Team = () => (
+  <section id="team" className="bg-[#cddecc]">
+    <div className="max-w-screen-2xl mx-auto">
+      <h1 className="pt-10 text-center max-sm:text-5xl">Team</h1>
+      <div className="grid grid-cols-1 md:grid-cols-3 overflow-auto flex-wrap gap-x-8 xl:gap-x-16 px-12">
```

Comment:
can you pls add sm:grid-cols-2, there are two cards for sm screen
<img width="771" alt="image" src="https://github.com/user-attachments/assets/b3ee20fc-4e74-4279-bb6e-67246f6fe8c8" />